docs/Algorithms.md

```diff
@@ -1,8 +1,35 @@
 # Implemented Compression Methods
 
-Each compression method receives its own hyperparameters that are organized as a dictionary and basically stored in a JSON file that is deserialized when the training starts. Compression methods can be applied separately or together producing sparse, quantized, or both sparse and quantized models. For more information about the configuration, refer to the samples.
+## Post-training Compression
 
-- [Quantization](./compression_algorithms/Quantization.md)
+- [Quantization](./usage/post_training_compression/post_training_quantization/Usage.md) (OpenVino, PyTorch, ONNX, TensorFlow)
```

Comment:
OpenVino -> OpenVINO. Please check the entire document for this typo.

Reply:
Done